libraries/parcelable/src/main/java/com/uber/okbuck/example/common/Person.java

```diff
@@ -1,8 +1,11 @@
 package com.uber.okbuck.example.common;
 
+import android.content.Context;
 import android.os.Parcel;
 import android.os.Parcelable;
 
+import com.uber.okbuck.example.parcelable.R;
```

Comment:
That's the one I'm using, `com.uber.okbuck.example.parcelable.R` is the merged R for this library